src/main/webapp/app/entities/share/share-visualization.tsx

```diff
@@ -0,0 +1,134 @@
+
+const ShareVisualization = (props: IShareVisualizationProps) => {
+
+    const params = new URLSearchParams(props.location.search);
+    const visualizationId = params.get('visualizationId');
+    const viewId = params.get('viewId');
+    const datasourceId = Number(params.get('datasourceId'));
+
+    useEffect(() => {
+        props.setIsShare(true);
+        if (visualizationId) {
+            props.receiveSocketResponse();
+            props.getVisualmetadataEntity(visualizationId);
+            props.getViewEntity(viewId);
+        }
+    }, []);
+
+    useEffect(() => {
+        if (props.visualmetadataEntity.fields && ValidateFields(props.visualmetadataEntity.fields)) {
+            props.receiveSocketResponseByVisualId(props.visualmetadataEntity.id);
+            const visualMetadata = VisualWrap(props.visualmetadataEntity);
+            const queryDTO = visualMetadata.getQueryParameters(props.visualmetadataEntity, {}, getConditionExpression({}), 0);
+            const body = {
+                queryDTO,
+                visualMetadata,
+                validationType: 'REQUIRED_FIELDS',
+                actionType: null,
+                type: 'share-link',
+            };
+            forwardCall(datasourceId, body, viewId);
+        }
+    }, [props.visualmetadataEntity]);
+
+    useEffect(() => {
+        if (props.visualDataById) {
+            if (props.visualDataById?.data.length > 0) {
+                renderVisualization(props.visualmetadataEntity, props.visualDataById?.data, 'visualization-edit', props);
+            }
+        }
+    }, [props.visualDataById]);
+
+    return (
+        <>
+            <FilterPanel {...props} />
+            <Flex direction="column" flex gap="size-75">
+
+                {/* <View >
+
+                    <Flex direction="row" gap="size-100" justifyContent={"space-between"}>
+                        <View>
+                            <span className="spectrum-Heading spectrum-Heading--sizeXXS" >
+                                Title : 
+                            </span>
+                            <span className="spectrum-Body spectrum-Body--sizeXS" >
+                                {props.visualmetadataEntity?.titleProperties?.titleText}
+                            </span>
+                        </View>
+                        <View>
+                            <HeaderIcon
+                                key={translate('views.menu.filter')}
+                                icon={<Filter size="M" />}
+                                title={translate('views.menu.filter')}
+                                onPress={props.toggleFilterPanel}
+                            />
+                        </View>
+                    </Flex>
+
+
+                </View> */}
+                <View >
+                    <div id={`visualization-edit-${props.visualmetadataEntity.id}`} className="visualization-share"></div>
```

Comment:
try to use View instead of View if possible

Reply:
not getting your point.